programs/stake/src/config.rs

```diff
@@ -28,9 +28,9 @@ pub fn create_account(lamports: u64, config: &Config) -> AccountSharedData {
 #[allow(deprecated)]
 pub fn add_genesis_account(genesis_config: &mut GenesisConfig) -> u64 {
     let mut account = create_config_account(vec![], &Config::default(), 0);
-    let lamports = genesis_config.rent.minimum_balance(account.data().len());
+    let lamports = std::cmp::max(genesis_config.rent.minimum_balance(account.data().len()), 1);
 
-    account.set_lamports(lamports.max(1));
+    account.set_lamports(lamports);
 
     genesis_config.add_account(solana_sdk_ids::config::id(), account);
```

Comment:
double value bp. this line is the bug that caused https://github.com/anza-xyz/agave/issues/6385 :joy: